build.gradle.kts

```diff
@@ -58,18 +58,12 @@ afterEvaluate {
 if (project.hasProperty("release.main")) {
```

Comment:
Can you actually remove this property check, and apply the plugin at the top with other plugins? This used to be a problem, but shouldn't be anymore since we removed the jreleaser config in `:smithy-cli`.